custom_components/tuya_local/devices/fisher_summer_air_conditioner.yaml

```diff
@@ -0,0 +1,463 @@
+      - id: 114
+        name: option
+        type: string
+        mapping:
+          - dps_val: 0
+            value: None
+            default: true
+          - dps_val: 1
+            value: Left and Right
+          - dps_val: 2
+            value: Left
+          - dps_val: 3
+            value: Middle
+          - dps_val: 4
+            value: Right
+          - dps_val: 5
+            value: Partial Left
+          - dps_val: 6
+            value: Partial Right
+          - dps_val: 7
+            value: Wide Angle
+  - entity: select
+    name: Energy Saving
+    category: config
+    dps:
+      - id: 119
+        name: option
+        type: string
+        mapping:
+          - dps_val: 0
+            value: None
+            default: true
+          - dps_val: 1
+            value: Default Power Saving
+          - dps_val: 2
+            value: Quota Power Saving
+          - dps_val: 3
+            value: Fixed Temperature Power Saving
+        optional: true
+  - entity: select
+    name: Generator Mode
+    category: config
+    dps:
+      - id: 120
+        name: option
+        type: string
+        mapping:
+          - dps_val: off
+            value: None
+            default: true
+          - dps_val: L1
+            value: L1
+          - dps_val: L2
+            value: L2
+          - dps_val: L3
+            value: L3
+        optional: true
+  - entity: sensor 
+    name: Boolcode Options
+    dps:
+      - id: 123
+        name: sensor
+        type: string
+  - entity: sensor
+    name: Air Quality
+    dps:
+      - id: 125
+        name: sensor
+        type: string
+        optional: true
+  - entity: select
+    name: Set Vertical Direction # Tuya JSON phrasing: Freeze Vertical
+    category: config
+    dps:
+      - id: 126
+        name: option
+        type: string
+        mapping:
+          - dps_val: 0
+            value: Freeze Current Position
+            default: true
+          - dps_val: 1
+            value: Top
+          - dps_val: 2
+            value: Slightly Up
+          - dps_val: 3
+            value: Middle
+          - dps_val: 4
+            value: Slightly Down
+          - dps_val: 5
+            value: Down  
+  - entity: select
+    name: Set Horizontal Direction # Tuya JSON phrasing: Freeze Horizontal
+    category: config
+    dps:
+      - id: 127
+        name: option
+        type: string
+        mapping:
+          - dps_val: 0
+            value: Freeze Current Position
+            default: true
+          - dps_val: 1
+            value: Left
+          - dps_val: 2
+            value: Slightly Left
+          - dps_val: 3
+            value: Middle
+          - dps_val: 4
+            value: Slightly Right
+          - dps_val: 5
+            value: Right  
+          - dps_val: 6
+            value: Wide Angle Left
+          - dps_val: 7
+            value: Wide Angle Right
+          - dps_val: 8
+            value: Wide Angle
+  - entity: select
+    name: Power
```

Comment:
Should this be configurable by the user? Is it not a fixed property of the particular model (which could just be an extra attribute) ?